### ISSUE TITLE
[BUG]-"odometry reading incorrect"

### ISSUE THREAD

**Issue by catr1xLiu** [Shenzhen-Robotics-Alliance/maple-sim#52]
## Bug Description
The odometry of the AdvancedSwerveDrive example project is not working correctly.

## Robot Code Repository
https://github.com/Shenzhen-Robotics-Alliance/maple-sim/tree/main/templates/AdvantageKit_AdvancedSwerveDriveProject

## Steps to Reproduce
Run the robot code and display the odometry positions

**Comment by catr1xLiu** [comment]
seems like an issue regarding the steer gear ratio.

```java
// in ModuleIOSim.java of the project
inputs.odometryTurnPositions = Arrays.stream(moduleSimulation.getCachedSteerRelativeEncoderPositions())
        .map(relativeEncoderPosition ->
                new Rotation2d(relativeEncoderPosition.divide(moduleSimulation.STEER_GEAR_RATIO)))
        .toArray(Rotation2d[]::new);
```

would fix it.

**Comment by catr1xLiu** [comment]
fixed in #54
